x11-misc/qt5ct/qt5ct-0.34.ebuild

```diff
@@ -28,7 +28,7 @@ DEPEND="${RDEPEND}
 "
 
 src_configure() {
-	eqmake5 DEFINES="$(usex dbus '' QT_NO_DBUS)" ${PN}.pro
+	eqmake5 DEFINES="$(usex dbus '' QT_NO_DBUS=1)" ${PN}.pro
```

Comment:
I think ``dbus`` USE flag is broken now.

From ``src/qt5ct-qtplugin/qt5ct-qtplugin.pro``:
```
!contains(DEFINES, QT_NO_DBUS):greaterThan(QT_MINOR_VERSION, 5) {
    QT += dbus
}
```

I believe this is operating based on Qt config, not what's passed to qmake. So, it will automagically link to qtdbus if installed.

Reply:
AFAIR from reading qmake docs is says `contains(DEFINES)` checks for cmd as well.
Also built qt5ct[-dbus] on a system with qtdbus and qtgui[dbus] and no linking happens according to `readelf -d`. Am I missing smth?